go/kbfs/libkbfs/disk_block_cache.go

```diff
@@ -849,6 +850,10 @@ func (cache *DiskBlockCacheLocal) Delete(ctx context.Context,
 	}
 
 	cache.log.CDebugf(ctx, "Cache Delete numBlocks=%d", len(blockIDs))
+	for _, bID := range blockIDs {
+		cache.log.CDebugf(ctx, "Cache type=%d delete block ID %s",
+			cache.cacheType, bID)
```

Comment:
This could be pretty verbose. But I guess we have other very verbose messages (e.g. `Put`)

Reply:
I found this useful when debugging this flake, but I would buy an argument that it's too verbose for normal logging.

Reply:
Yeah I'd prefer not to have this on by default, though I agree it's useful in tests.  Maybe wrap the `for` loop with `if cache.Mode().IsTestMode()`?